frontend/public/components/pod.tsx

```diff
@@ -551,7 +552,49 @@ const PodGraphs = requirePrometheus(({ pod }) => {
   );
 });
 
-export const PodStatus: React.FC<PodStatusProps> = ({ pod }) => <Status status={podPhase(pod)} />;
+const PodStatusPopover: React.FC<PodStatusPopoverProps> = ({
+  bodyContent,
+  headerContent,
+  status,
+}) => {
+  return (
+    <Popover headerContent={headerContent} bodyContent={bodyContent}>
+      <Button variant="link" isInline>
+        <Status status={status} />
+      </Button>
+    </Popover>
+  );
+};
+
+export const PodStatus: React.FC<PodStatusProps> = ({ pod }) => {
+  const status = podPhase(pod);
+  const unschedulableCondition = pod.status?.conditions?.find(
+    (condition) => condition.reason === 'Unschedulable' && condition.status === 'False',
+  );
+  const containerStatusStateWaiting = pod.status?.containerStatuses?.find(
+    (cs) => cs.state?.waiting,
+  );
+
+  if (status === 'Pending' && unschedulableCondition) {
+    return (
+      <PodStatusPopover
+        bodyContent={unschedulableCondition.message}
+        headerContent={`${PodModel.label} unschedulable`}
```

Comment:
Switched this to use the model.label so at least one of the English words is from a single source.  I assume we want all this content in English since the content from the API is English.

Reply:
In general, I'd err on the side of externalizing the string. FWIW, I think in the other API doc popovers we show the translated property name with the English API doc string.